.ci/Jenkinsfile

```
@@ -250,6 +250,37 @@ pipeline {
         }
       }
     }
+
+    /**
+    Daily snapshots.
+    */
+    stage('Daily snapshots') {
```

Is it okay to change the name from "Daily snapshots", because this isn't really "Daily"?

```suggestion
    stage('Publish snapshot') {
```